### ISSUE TITLE
Error when concatenating empty directory

### ISSUE THREAD

**Issue by jme783** [mariocasciaro/gulp-concat-css#22]
With the following code: 
```
gulp.task('vendor-css', function() {
  gulp.src('./vendor/css/*.css')
    .pipe(concatCss('vendor.css'))
    .pipe(gulp.dest('./public/styles'));
});
```
And my vendor/css directory empty, I receive this error when trying to run the task:

`TypeError: Cannot read property 'base' of undefined`

I think the module requires that there are actually css files where you are telling it to concatenate. Ideally, it is flexible enough to skip the task if there are no CSS files present, and avoid throwing this error! For now, I'll put a placeholder CSS file there to avoid this.

Thanks!

**Comment by mariocasciaro** [comment]
Landed in 2.1.2. Thanks for reporting.

**Comment by jme783** [comment]
:+1:
